tools/addrundata/rundata.go

```diff
@@ -91,29 +122,22 @@ func (pd *parsedData) parseInit(sc *bufio.Scanner) error {
 	return errors.New("func init() was not terminated")
 }
 
-var tmpl = template.Must(template.New("rundata_test.go").Parse(
-	`// Code generated by go run tools/addrundata; DO NOT EDIT.
-package {{.Package}}
-
-import "github.com/openconfig/featureprofiles/internal/rundata"
+var tmpl = template.Must(template.New("metadata.textproto").Parse(
+	`# proto-file: proto/metadata.proto
+# proto-message: Metadata
 
-func init() {
-{{range .Data}}	rundata.{{.Key}} = {{printf "%q\n" .Value}}{{end -}}
-}
+uuid: "{{.UUID}}"
+plan_id: "{{.PlanID}}"
+description: "{{.Description}}"
 `))
 
-// write generates a complete rundata_test.go to the writer.
-func (pd *parsedData) write(w io.Writer, pkg string) error {
-	tmpl.Execute(w, &struct {
-		Package string
-		Data    []struct{ Key, Value string }
+// write generates a complete metadata.textproto to the writer.
```

Comment:
Request change: we cannot simply overwrite metadata with only these fields. If there are other fields present in the metadata (e.g. testbed, config and telemetry coverage), we have to preserve them.